PhysicsTools/PatUtils/python/tools/runMETCorrectionsAndUncertainties.py

```diff
@@ -242,20 +261,22 @@ def __call__(self, process,
         else:
              #internal jet configuration
             self.jetConfiguration()
+
+        #defaults for 2017 fix
+        #(don't need to recluster, just uses a subset of the input jet coll)
+        if fixEE2017:
+            if recoMetFromPFCsIsNone: self.setParameter('recoMetFromPFCs',True)
+            if reclusterJetsIsNone: self.setParameter('reclusterJets',False)
         
         #met reprocessing and jet reclustering
-        if recoMetFromPFCs:
+        if recoMetFromPFCs and reclusterJetsIsNone and not fixEE2017:
             self.setParameter('reclusterJets',True)
         
         #ZD: puppi jet reclustering breaks the puppi jets
         #overwriting of jet reclustering parameter for puppi
         if self._parameters["Puppi"].value and not onMiniAOD:
             self.setParameter('reclusterJets',False)
 
-        #jet collection overloading for automatic jet reclustering or JEC application
```

Comment:
This was not removed in #24876, or later in the master: please check and fix

Reply:
Hi @perrotta I didn't understand exactly what is this problem. Could you please help me on this?. I didn't remove this line in 94X.

Reply:
These lines are not there any more: therefore, someone did remove them.

It was done by this commit from @kpedro88 :
https://github.com/cms-sw/cmssw/commit/4c2621a03feb384122d9597e78d681ba8759bbf2#diff-496fcf98e9a0901abbe07acea8ebda40

The same commit on the master only removed one empty line:
https://github.com/cms-met/cmssw/commit/6503137e8e9f9021f1cbdc17622d38be160e3dba#diff-496fcf98e9a0901abbe07acea8ebda40

Please find out together with Kevin what happened there, and provide a fix either in 94X or in the master (or both...)

Reply:
These lines were removed deliberately, because this default value was not actually used (overridden later), and its presence was confusing. It should have been removed in the upstream PR also. Perhaps there was a mistake in the rebase.

Reply:
This makes sense. Please provide a PR in the master with the same removal then, so that the backport will remain synchronized with it.